.github/scripts/setup_runners_service.sh

```diff
@@ -136,6 +140,8 @@ then
     sleep 30
     kill -9 `ps -ef | grep [e]lasticsearch | awk '{print $2}'`
     sed -i /^node.max_local_storage_nodes/d ./config/elasticsearch.yml
+    # Increase the number of allowed script compilations. The SQL integ tests use a lot of scripts.
+    sudo echo "script.context.field.max_compilations_rate: 1000/1m" >> ./config/elasticsearch.yml
```

Comment:
Why not add it to line 109, this section is the startup of the server, not the config part.

Reply:
Yeah, I wondered about that. I was following the pattern where the "node.max_local_storage_nodes" line is deleted (line 142, at startup for zip, but at config time for docker/other).

Should the node.max_local_storage_nodes thing be moved up as well?

Reply:
These two should not related, if your line do not require a startup of the server once to take into effect, then just put it in 109 so all the rest of the steps can consume it.